util/sleep_linux.go

```diff
@@ -14,6 +14,9 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
 
+//go:build !arm && linux
+// +build !arm,linux
```

Comment:
the _linux filename provides its own build tag constraints based on whether GOOS=linux ... I'm not sure if you can mixing file suffixes with build tags? or maybe you can?
https://dave.cheney.net/2013/10/12/how-to-use-conditional-compilation-with-the-go-build-tool

if not maybe change the name of this file to not end with _linux (but otherwise keep the rest the same)

Reply:
oh
>If a file's name, after stripping the extension and a possible _test suffix, matches any of the following patterns:
>*_GOOS
>*_GOARCH
>*_GOOS_GOARCH
>(example: source_windows_amd64.go) where GOOS and GOARCH represent any known operating system and architecture values respectively, then the file is considered to have an implicit build constraint requiring those terms (in addition to any explicit constraints in the file).

^^ "in addition to any explicit constraints in the file" OK!